package.json

```diff
@@ -15,15 +15,18 @@
         "artedi": "1.1.1",
         "assert-plus": "0.1.5",
         "backoff": "2.3.0",
-        "boray": "git+https://github.com/joyent/node-boray.git#master",
+        "bignum": "0.6.0",
+        "boray": "git+https://github.com/joyent/node-boray.git#MANTA-4604",
```

Comment:
This perhaps should change since `node-boray` has its own PR

Reply:
Let's just wait and merge the `node-boray`/`node-buckets-mdapi` PR and then update this  right before merging.

Reply:
Addressed, it's pointing to `master` now which is itself also a temporary condition.